tests/test.py

```diff
@@ -1222,9 +1267,15 @@ def tearDown(self):
         processes = [getattr(self, 'connector_1', ProcessStub()),
                      getattr(self, 'connector_2', ProcessStub()),
                      getattr(self, 'acra', ProcessStub())]
+
+        if self.with_tls():
+            processes.append(getattr(self, 'ocsp_server', ProcessStub()))
+
         stop_process(processes)
         send_signal_by_process_name('acra-server', signal.SIGKILL)
         send_signal_by_process_name('acra-connector', signal.SIGKILL)
+        if self.with_tls():
+            send_signal_by_process_name('openssl', signal.SIGKILL)
```

Comment:
Hm... While it might be okay to blanket send SIGKILLs to `acra-*` things, doing so with all `openssl` instances is a bit risky.

@Lagovas, do you remember why these `send_signal_by_process_name()` calls are necessary? `stop_process()` already attempts to do graceful termination with SIGTERM and then issues a SIGKILL after a timeout, sending them out to specified PIDs, not just to all processes with a matching name.

Reply:
actually some time we had issues with zombi processes after tests, when forked processes didn't stop by sigterm or some test cases incorrectly sent sigkill (forgot or exited before sigkill call)